Assets/Scripts/Game/CombatManager.cs

```diff
@@ -131,4 +134,45 @@ public void UpdateBossState(bool _state)
     {
         isBossAlive = _state;
     }
+    public void UpdateCountOfEachEnemyType(int _delta, int _enemyType)
+    {
+        numberOfEachActiveEnemyType[_enemyType] += _delta;
+    }
+    public EntityBehaviour.enemyType CountHighestEnemy()
+    {
+        int _highest = 0;
+        int _enemyChoice = 0;
+        for (int i =0; i < 6; i++)
+        {
+            _highest = Mathf.Max(_highest, numberOfEachActiveEnemyType[i]);
+            //Debug.Log("i: " + i);
+            //Debug.Log("currenthighest: " + _highest);
+        }
+        //Debug.Log("final highest: " + _highest);
```

Comment:
consider removing commented-out debug lines, unless you know for a fact there is *good reason* to put them on master

Reply:
removed